src/package1/package1Version.ts

```diff
@@ -172,3 +136,32 @@ export class Package1Version implements IPackageVersion1GP {
     return (await this.connection.tooling.query<PackagingSObjects.MetadataPackageVersion>(query)).records;
   }
 }
+
+const packageUploadPolling = async (
```

Comment:
I believe this is only applicable to 1GP packages, and why it was kept under the `Package1Version` class.  I think keeping that clarity to the consumer is good

Reply:
I'm not following.  it's not exported, so there shouldn't be any consumers outside the package1Version module, right?